Cabal/tests/ParserTests/regressions/nothing-unicode.golden

```diff
@@ -0,0 +1,24 @@
+name: 無
+version: 0
+cabal-version: >=1.10
+build-type: Simple
+license: UnspecifiedLicense
+synopsis: The canonical non-package 無
+x-無: 無
```

Comment:
custom fields can be, but I added a check that we don't want them on  Hackage.

The story is very similar as with flag: French or German shop might have some `x-Straße` there. Let them have it, if they want.